plugin/code_actions.py

```diff
@@ -74,7 +74,8 @@ def response_filter(session: Session, actions: List[CodeActionOrCommand]) -> Lis
             if only_kinds:
                 return [a for a in actions if not is_command(a) and is_kinds_include_kind(only_kinds, a.get('kind'))]
             return [
-                a for a in actions if is_command(a) or is_kinds_include_kind([CodeActionKind.QuickFix], a.get('kind'))
+                a for a in actions
+                if is_command(a) or not a.get('kind') or is_kinds_include_kind([CodeActionKind.QuickFix], a.get('kind'))
```

Comment:
The `is_command(a)` is technically not needed.

Ideally the `is_command` type guard would tell pyright/mypy that `a` in later conditions is not of `Command` type but they are not that smart so it's still an union. But at least pyright doesn't complain about `.get('kind')` (`mypy` does but we don't care right now).